app/helpers/plantings_helper.rb

```diff
@@ -1,13 +1,15 @@
 module PlantingsHelper
   def display_days_before_maturity(planting)
     if planting.finished?
-      0
+      "0"
     elsif !planting.finished_at.nil?
-      ((p = planting.finished_at - DateTime.now).to_i) <= 0 ? 0 : p.to_i
+      ((p = planting.finished_at - Date.today).to_i) <= 0 ? "0" : p.to_i.to_s
```

Comment:
rubocop is grumpy about `Date.today` without a timezone. Perhaps you want `Date.current`?